### PR TITLE
SAK-44150: samigo > add button to copy published assessment url to clipboard (a11y)

### PR DESCRIPTION
**Pull request by bjones86**
https://jira.sakaiproject.org/browse/SAK-44150

Keyboard users are unable to select this text for copy/paste. Suggestion from accessibility expert is to add a copy to clipboard button.

### CODE REVIEW

**Comment by bjones86** [comment]
This depends on #8350

File: samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp [hunk 1]
```diff
@@ -245,6 +250,10 @@
                         styleClass="form-control-label col-md-2"/>
         <div class=" col-md-10">
             <h:outputText value="#{publishedSettings.publishedUrl}" />
+            <button type="button" id="copyToClipboard" title="<h:outputText value='#{assessmentSettingsMessages.copyToClipboard}' />">
+                <span class="fa fa-clipboard" aria-hidden="true"></span>
+                <span class="hide"><h:outputText value="#{assessmentSettingsMessages.copyToClipboard}" /></span>
```
Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp:255]: Is this meant to be hidden from everyone except screen readers? If yes, please just use sr-only instead

Comment (bjones86) [reply samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp]: Yes I was just in the process of changing that.

**Review by ern — APPROVED** [review]
Thanks for the update!

**Pull request merged**